microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -106,41 +119,42 @@ The following are some of the flags we've seen used.
 | `msEdgeDevToolsWdpRemoteDebugging` | Enables remote debugging.  See [Remote debugging WebView2 WinUI 2 (UWP) apps](../how-to/remote-debugging.md), [Remotely debug Windows devices](../../devtools-guide-chromium/remote-debugging/windows.md). |
-| `msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs` | If this flag is enabled, the `AddWebResourceRequestedFilter` method (including overloads) will also apply to out-of-process iframes. <br/>.NET: [AddWebResourceRequestedFilter](/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter) <br/>WinRT: [AddWebResourceRequestedFilter](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#addwebresourcerequestedfilter) <br/>Win32: [AddWebResourceRequestedFilterWithRequestSourceKinds](/microsoft-edge/webview2/reference/win32/icorewebview2_22#addwebresourcerequestedfilterwithrequestsourcekinds) |
+| `msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs` | Makes the `AddWebResourceRequestedFilter` method (including overloads) also apply to out-of-process iframes. <br/>.NET: [AddWebResourceRequestedFilter](/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter) <br/>WinRT: [AddWebResourceRequestedFilter](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#addwebresourcerequestedfilter) <br/>Win32: [AddWebResourceRequestedFilterWithRequestSourceKinds](/microsoft-edge/webview2/reference/win32/icorewebview2_22#addwebresourcerequestedfilterwithrequestsourcekinds) |
 | `msEnhancedTextContrast` | Improves text contrast enhancement and gamma correction to match the quality and clarity of other native Windows applications.  When this flag is used, font rendering respects user ClearType Tuner settings when applying text contrast enhancement and gamma correction. |
 | `msEnhancedTrackingPreventionEnabled` | Enables native privacy protection features, such as blocking cookies and web requests that reside in domains that are known to be tracking domains. |
-| `msFloatyMode` | If `false`, disables the Floaty feature.  Use this flag to disable the Floaty feature, because WebView doesn't support browser retention experiments. |
-| `msFloatyShouldHonorIndiaHoldout` | If `true`, honors the India holdout group.  Use this flag, set to `false`, to disable the Floaty feature that's enabled if the user is part of the India holdout group, because WebView doesn't support browser retention experiments. |
+| `msFloatyMode` | Enables the Floaty feature.  If not enabled, the Floaty feature is disabled; for example because WebView doesn't support browser retention experiments. |
+| `msFloatyShouldHonorIndiaHoldout` | If enabled, honors the India holdout group.  If not enabled, disables the Floaty feature that's enabled if the user is part of the India holdout group, because WebView doesn't support browser retention experiments. |
```

Comment:
```suggestion
| `msFloatyShouldHonorIndiaHoldout` | If enabled, honors the India holdout group.  If not enabled, disables the Floaty feature that's enabled if the user is part of the India holdout group, because WebView doesn't support browser retention experiments.  Takes a `true`|`false` value. |
```